controller.py

```diff
@@ -416,7 +417,7 @@ def updates_feed(self, metadata_identifier):
         last_update_time = flask.request.args.get('last_update_time', None)
         if last_update_time:
             try:
-                last_update_time = datetime.strptime(
+                last_update_time = strptime_utc(
                     last_update_time, self.TIMESTAMP_FORMAT
```

Comment:
While looking at this I discovered that my core branch is wrong. The  "Z" in the TIMESTAMP_FORMAT doesn't describe a timezone-naive datetime, although the Atom RFC seems to say that's what it's for. The "Z" means Zulu or Zero time, i.e. UTC. That's what I thought originally.
 
I was going to say we need to get rid of the "Z" in timestamps whenever we're communicating across machines, but this is correct. The flip side is that if we do get a naive datetime there's no way to represent that in Atom -- best we can do is pretend it's UTC.

Reply:
When you say "but this is correct", do you mean that leaving the "Z" in the timestamp is correct?

Looking at this again, I just realized that `TIMESTAMP_FORMAT` is missing `%z` in the string. That's what picks up the UTC offset in an ISO-8601 date string. The "Z" should be equivalent to an UTC offset so I think it's equivalent to have either "Z" or "+00:00", right? If we do leave the "Z", I think it's the default +00:00 without an offset but _that_ may be wrong.

Reply:
Yeah, leaving "Z" in TIMESTAMP_FORMAT is correct _if_ the time is UTC. Otherwise you have to have a numeric offset (or, I guess, omit timezone information altogether).

`Z` is equivalent to `+00:00` and a given timestamp should only have one of them.